pkg/schema/schema_reader.go

```diff
@@ -0,0 +1,42 @@
+package schema
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+)
+
+func ReadConfigFromPath(path string) (*SchemaJson, error) {
+
+	extension := filepath.Ext(path)
+	isYaml := isYamlExtension(extension)
+	if extension != ".json" && !isYaml {
+		return nil, fmt.Errorf("unsupported extension \"%s\" found in path \"%v\". Only .json, .yaml and .yml are supported", extension, path)
+	}
+
+	var bytes []byte
+	bytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
```

Comment:
I like to see errors [wrapped](https://blog.golang.org/go1.13-errors#TOC_3.3.). For instance in this case the user would just see failed to read file error but it might not indicate that it's the obfuscation configuration file.